src/server/game/Entities/Object/Object.cpp

```diff
@@ -1500,10 +1500,10 @@ float WorldObject::GetGridActivationRange() const
     {
         if (ToPlayer()->GetCinematicMgr()->IsOnCinematic())
             return DEFAULT_VISIBILITY_INSTANCE;
-        return GetMap()->GetVisibilityRange();
+        return GetMap()->GetVisibilityRange() * 5;
```

Comment:
Implementation aside, are there any big reasons against having players activate nearby grids rather than only the one they're currently in?

Would solve this issue as well as make waypoint movement more realistic (right now every npc only starts movement when someone can see them).

Could be a configurable option, since it would more than double the amount of grids loaded. Opinions?